File: CSharp/OpenTraceability/Models/Events/PersistentDisposition.cs
using OpenTraceability.Utility.Attributes;

namespace OpenTraceability.Models.Events
{
    public class PersistentDisposition
    {
        [OpenTraceability("unset", 1)]
        [OpenTraceabilityArray]
        public List<Uri>? Unset { get; set; }

        [OpenTraceability("set", 2)]
        [OpenTraceabilityArray]
        public List<Uri>? Set { get; set; }
    }
}

File: JDK11/opentraceability/src/main/java/opentraceability/models/events/persistentdisposition.java
package opentraceability.models.events;

import java.net.URI;
import opentraceability.utility.attributes.OpenTraceabilityArrayAttribute;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;

public class PersistentDisposition {
    @OpenTraceabilityAttribute("", "unset", 1)
    @OpenTraceabilityArrayAttribute
    public MutableList<URI> unset = null;

    @OpenTraceabilityAttribute("", "set", 2)
    @OpenTraceabilityArrayAttribute
    public MutableList<URI> set = null;
}